mypy/typeshed/stdlib/re.pyi

```diff
@@ -74,19 +74,19 @@ class Match(Generic[AnyStr]):
     @overload
     def group(self, __group: Literal[0] = 0) -> AnyStr: ...
     @overload
-    def group(self, __group: str | int) -> AnyStr | Any: ...
+    def group(self, __group: str | int) -> AnyStr | None: ...
```

Comment:
are these changes also being added to basedtypeshed

Reply:
Thanks, currently the implementation of basedtypeshed is a but ruff. I'll fix this up there.